custom/inddex/ucr_data.py

```diff
@@ -37,23 +38,42 @@ def table_name(self):
     @property
     def filters(self):
         filters = [GTE('visit_date', 'startdate'), LTE('visit_date', 'enddate')]
-        for column in self.FILTERABLE_COLUMNS:
+        if self._age_range:
+            filters.append(self._get_age_range_filter())
+        if self.config.get('owner_id'):
+            infilter_bindparams = get_INFilter_bindparams('owner_id', self.config['owner_id'])
+            filters.append(IN('owner_id', infilter_bindparams))
+        for column in [
```

Comment:
You could DRY this a bit by defining
```python
EQ_FILTERABLE_COLUMNS = [
    'breastfeeding',
    'gender',
    'pregnant',
    'recall_status',
    'supplements'
    'urban_rural'
]
```
and preserving the value of `FILTERABLE_COLUMNS` for the food module with:
```python
FILTERABLE_COLUMNS = [
    'age_range',
    'owner_id',
] + EQ_FILTERABLE_COLUMNS
```